src/test/java/com/playground/playground/controller/MlParametersControllerTest.java

```diff
@@ -0,0 +1,82 @@
+package com.playground.playground.controller;
+
+import com.playground.playground.interface_adapter.controller.MlParametersController;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import org.junit.jupiter.api.Test;
+import org.testfx.framework.junit5.ApplicationTest;
+
+import static org.junit.Assert.assertEquals;
+
+class MlParametersControllerTest extends ApplicationTest {
```

Comment:
No javadocs

Reply:
Wasn't provided for other tests, I'll change that upon merge.